cves/CVE-2014-0098.yml

```diff
@@ -142,10 +149,15 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 577eaf25cdc37727001d83a858b75c11687ad745
+      note: |
+        This commit reverts a previous commit that affects the method containing
```

Comment:
As an outsider, this note made little sense to me. One large run-on sentence. Break it up into a few sentences, and it would read infinitely better

Reply:
Hopefully made this more readable, as well as fixed it up based on your other suggestions. Thanks!